lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -316,7 +316,8 @@ def execute
       end
 
       def create_tar(cvv, repositories, json)
-        export_prefix = "export-#{cvv['id']}"
+        cv_name = cvv['name'].gsub(/\s.+/, '')
```

Comment:
CVVs don't have names. They have versions and ids.

Reply:
You're going to have to pass in the CV as well as the CVV to this method to achieve what you want.

Reply:
@akofink 

Something like this?

```
        cv = show(:content_views, 'id' => cvv['content_view_id'])
        export_prefix = "export-#{cv['name']}-#{cvv['id']}"
```

That got me the expected result.

Reply:
the 'name' field on the cvv var is from ` cvv = show(:content_view_versions, 'id' => options['option_id'])`. I'm guessing its just pulling in the CV name from that api call.

However, it's probably better to pass in both cvv and cv into `create_tar`, just so its more clear what's happening. `cv` might also have a label so you don't need to use the gsub :cat: